### PR TITLE
docs: add web release notes

### PR DESCRIPTION
**Pull request by amar-1995**

### CODE REVIEW

File: docs/javascript/v2/changelog/release-notes.mdx [hunk 1]
```diff
@@ -15,6 +15,18 @@ description: Release Notes for 100ms JavaScript SDK
...
+## 2023-09-13
+Released: `@100mslive/hms-video-store@0.10.15`, `@100mslive/react-sdk@0.8.15`, `@100mslive/hls-player@0.1.15`, `@100mslive/roomkit-react@0.1.6`
+
+### Important
+
+We released Prebuilt. Customize your app themes and add video conferencing to your app with one line of code.
+Details [here](https://www.100ms.live/docs/javascript/v2/quickstart/prebuilt-quickstart)
+
+### Fixed: 
+- Showing the correct video aspect ratio for mobile web.
```
Comment (saikatmitra91) [inline docs/javascript/v2/changelog/release-notes.mdx:27]: mention the package at the end

File: docs/javascript/v2/changelog/release-notes.mdx [hunk 2]
```diff
@@ -15,6 +15,18 @@ description: Release Notes for 100ms JavaScript SDK
...
+- Screenshare track was showing presenter video track.
```
Comment (saikatmitra91) [inline docs/javascript/v2/changelog/release-notes.mdx:28]: mention package at the end

File: docs/javascript/v2/changelog/release-notes.mdx [hunk 3]
```diff
@@ -15,6 +15,18 @@ description: Release Notes for 100ms JavaScript SDK
 | @100mslive/hms-video-react(deprecated) | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhms-video-react.svg)](https://badge.fury.io/js/%40100mslive%2Fhms-video-react) |
 
 
+## 2023-09-13
+Released: `@100mslive/hms-video-store@0.10.15`, `@100mslive/react-sdk@0.8.15`, `@100mslive/hls-player@0.1.15`, `@100mslive/roomkit-react@0.1.6`
+
+### Important
+
+We released Prebuilt. Customize your app themes and add video conferencing to your app with one line of code.
```
Comment (saikatmitra91) [inline docs/javascript/v2/changelog/release-notes.mdx:23]: Prebuilt was also available before. This version now supports changing themes and behaviour using dashboard.

**Review by saikatmitra91 — CHANGES_REQUESTED** [review]

File: docs/javascript/v2/changelog/release-notes.mdx [hunk 4]
```diff
@@ -15,6 +15,18 @@ description: Release Notes for 100ms JavaScript SDK
...
+Released: `@100mslive/hms-video-store@0.10.15`, `@100mslive/react-sdk@0.8.15`, `@100mslive/hls-player@0.1.15`, `@100mslive/roomkit-react@0.1.6`
+
+### Important
+
+Prebuilt is now updated. Now supports changing themes and behaviour using dashboard.
+Details [here](https://www.100ms.live/docs/javascript/v2/quickstart/prebuilt-quickstart)
+
+### Fixed: 
+- Showing the correct video aspect ratio for mobile web in @100mslive/hms-video-web@0.10.15
+- Screenshare track was showing presenter video track, fix are in @100mslive/hms-video-web@0.10.15
```
Comment (saikatmitra91) [inline docs/javascript/v2/changelog/release-notes.mdx:28]:
```suggestion
- Screenshare track was showing presenter video track - `@100mslive/hms-video-web`
```

File: docs/javascript/v2/changelog/release-notes.mdx [hunk 5]
```diff
@@ -15,6 +15,18 @@ description: Release Notes for 100ms JavaScript SDK
...
+## 2023-09-13
+Released: `@100mslive/hms-video-store@0.10.15`, `@100mslive/react-sdk@0.8.15`, `@100mslive/hls-player@0.1.15`, `@100mslive/roomkit-react@0.1.6`
+
+### Important
+
+Prebuilt is now updated. Now supports changing themes and behaviour using dashboard.
+Details [here](https://www.100ms.live/docs/javascript/v2/quickstart/prebuilt-quickstart)
+
+### Fixed: 
+- Showing the correct video aspect ratio for mobile web in @100mslive/hms-video-web@0.10.15
```
Comment (saikatmitra91) [inline docs/javascript/v2/changelog/release-notes.mdx:27]:
```suggestion
- Capture video at correct aspect ratio for mobile web - `@100mslive/hms-video-web`
```

**Review by saikatmitra91 — DISMISSED** [review]

File: docs/javascript/v2/changelog/release-notes.mdx [hunk 6]
```diff
@@ -15,6 +15,18 @@ description: Release Notes for 100ms JavaScript SDK
...
+Released: `@100mslive/hms-video-store@0.10.15`, `@100mslive/react-sdk@0.8.15`, `@100mslive/hls-player@0.1.15`, `@100mslive/roomkit-react@0.1.6`
+
+### Important
+
+Prebuilt is now updated. Now supports changing themes and behaviour using dashboard.
+Details [here](https://www.100ms.live/docs/javascript/v2/quickstart/prebuilt-quickstart)
+
+### Fixed: 
+- Capture video at correct aspect ratio for mobile web - `@100mslive/hms-video-web`
+- Screenshare track was showing presenter video track - `@100mslive/hms-video-web`
```
Comment (raviteja83) [inline docs/javascript/v2/changelog/release-notes.mdx:28]: this was on role change right?

Comment (amar-1995) [reply docs/javascript/v2/changelog/release-notes.mdx:28]: no, We we stop screenshare, presenter video is shown

**Review by raviteja83 — APPROVED** [review]

**Pull request merged**